tests/testthat/test-create_landscape_covariates.R

```diff
@@ -0,0 +1,12 @@
+# test_that("create_landscape_covariates fails with invalid input", {
+#
+#   outdir <- withr::local_tempdir()
+#   sp <- "C:/Programs/saga-9.2.0_x64/saga-9.2.0_x64/saga_cmd.exe"
+#   aoi_snapped <- make_test_aoi(outdir)
+#   aoi_rast <- create_template_raster(aoi_snapped, res = 50, out_dir = outdir)
+#
+#   expect_error(create_landscape_covariates(aoi = 1, saga_path = sp))
+#   expect_error(create_landscape_covariates(aoi =  aoi_rast , saga_path = NULL))
+#
+# })
+
```

Comment:
I know they're not actually running yet, but some tests to test the functionality will be necessary I think (i.e., not just errors)

Reply:
You could even just put them in as placeholders for now until we get the SAGA detection working